plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/local-pickup-select/index.tsx

```
@@ -39,9 +58,55 @@ export const LocalPickupSelect = ( {
+	// We sometimes don't want to show items even if we have multiple packages.
+	const shouldShowItems = showItems ?? multiplePackages;
+
+	let header = multiplePackages && title && <div>{ title }</div>;
+
+	// packageData was added in version 10.4
+	if ( ( multiplePackages && packageData ) || shouldShowItems ) {
+		header = (
+			<div className="wc-block-components-shipping-rates-control__package-header">
+				<div
+					className="wc-block-components-shipping-rates-control__package-title"
+					dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)